content/release-10-17-0/platform-10-17-0-bundle/platform-10-17-0-364.md

```diff
@@ -0,0 +1,56 @@
+---
+weight: 44
+title: 10.17.0.364
+layout: bundle
+---
+
+<!--10.17.0.347 - 10.17.0.364-->
+
+### Fixes
+
+<table>
+<colgroup>
+<col style="width: 15%;">
+<col style="width:50%;">
+<col style="width: 10%;">
+<col style="width: 12%;">
+<col style="width: 13%;">
+</colgroup>
+<thead><tr>
+<th>
+Product area</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+<th>
+Build version</th>
+<th>Build comp.</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Administration</td>
+<td>Microservices which have been created via API without providing a binary for it can again be subscribed without getting an error message.</td>

```

Comment:
is "can again be subscribed" correct? it looks like there's a "to" missing, but i'm not sure if that was the intended meaning.

Reply:
We often have these to versions: "A microservice/application is subscribed", but "a tenant subscribes to a microservice/application". So Id say it is correct here.